### DISCUSSION TITLE
Open a project directly on the command line

### DISCUSSION THREAD

**Discussion by superuser7777** [alefragnani/vscode-project-manager#653] [Q&A] [answered]
I often use the launcher in a Windows environment.
Can I open a project directly from the command line?
We are currently forcing the switch with AHK & RPA.
Is there any demand?

**Comment by alefragnani** [comment, marked as answer]
Hi @superuser7777 ,

Unfortunately not. There is not tool/option for that and up until today, no demand either. 

But, I'm not sure this would be possible within the extension, or as an external tool (CLI). Based on what I see in the VS Code API, I suppose it would be an external tool.

Hope this helps.

**Reply by superuser7777** [reply]
Thank you for your reply.
Was self resolved.
I can achieve it by opening it with code.exe "project-manager-select-folder".
